.gitignore

```diff
@@ -26,3 +26,5 @@ historicalData/plotters/__pycache__
 0xb1125ba5878cf3a843be686c6c2486306f03e301_2.csv
 0xb1125ba5878cf3a843be686c6c2486306f03e301.csv
 deployments/mainnetForkDeployment.json
+Pipfile
```

Comment:
I ended up with a whole new virtual env directory when I did plotting. It sounds like there's a better way (I'm not familiar with `pipfile`s) so perhaps you could document what you think is best practice here?